packages/flutter_tools/lib/src/build_runner/build_script.dart

```diff
@@ -461,8 +463,32 @@ Future<void> bootstrapDart2Js(BuildStep buildStep, String flutterWebSdk, bool pr
   final AssetId jsOutputId = dartEntrypointId.changeExtension(jsEntrypointExtension);
   final File jsOutputFile = scratchSpace.fileFor(jsOutputId);
   if (result.succeeded && jsOutputFile.existsSync()) {
-    log.info(result.output);
-    // Explicitly write out the original js file and sourcemap.
+    final String rootDir = path.dirname(jsOutputFile.path);
+    final String dartFile = path.basename(dartEntrypointId.path);
+    final Glob fileGlob = Glob('$dartFile.js*');
+    final Archive archive = Archive();
+    await for (FileSystemEntity jsFile in fileGlob.list(root: rootDir)) {
```

Comment:
`.whereType<File>()` would remove the need for extra type checks inside the loop.

Reply:
Interestingly `Stream` doesn't define a `whereType`